src/Entity/Reporting/TeamMemberHistory.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace App\Entity\Reporting;
+
+use App\Entity\Adherent;
+use App\Entity\Administrator;
+use App\Entity\Team\Team;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(
+ *     name="team_member_histories",
```

Comment:
We used to apply the singular for the table name:
```suggestion
 *     name="team_member_history",
```